src/main.cpp

```diff
@@ -476,12 +484,27 @@ static void _assert_permissions(const string &sota_config_dir)
 static void _assert_not_registered(const string &sota_config_dir)
 {
 	string path = sota_config_dir + "/sql.db";
-	if (access(path.c_str(), F_OK ) == 0 ) {
+	if (_exists(path)) {
 		cerr << "ERROR: Device appears to already be registered in " << path << endl;
 		exit(EXIT_FAILURE);
 	}
 }
 
+static void _force(const string &db_path)
+{
+	cout << "Stopping aktualizr-lite" << endl;
+	_spawn("systemctl stop aktualizr-lite.path");
+	_spawn("systemctl stop aktualizr-lite.service");
```

Comment:
Do we also need to stop .path or would .service already be enough? The service will automatically run as a consequence of .path after registering, so wonder if we also need to manually start the job here (because we don't start on new registrations).

Reply:
Yes, I ran into this. `systemctl stop aktualizr-lite` is not enough. I've recorded what happens if you don't disable both services here: https://asciinema.org/a/PxSC6hcVRkeFB8IsRAphfunaR